app/views/leaderboard.scala.html

```diff
@@ -243,8 +243,8 @@ <h1 class="leaderboard-header weekly-header">@Messages("leaderboard.weekly.title
     }
     @if(user && user.get.role.getOrElse("") != "Anonymous" && !userOrg.isEmpty) {
         <div class="item leaderboard-table">
-            <h1 class="leaderboard-header">@Messages("leaderboard.org.title", orgName)</h1>
-            <h5>@Messages("leaderboard.org.detail", orgName)</h5>
+            <h1 class="leaderboard-header">@Messages("leaderboard.org.title", OrganizationTable.getOrganizationName(UserOrgTable.getOrg(user.get.userId).get).getOrElse("Team name not found"))</h1>
```

Comment:
Not sure if this was a merge conflict or something, but shouldn't we still be using the `orgName` variable defined at the top?